package.json

```diff
@@ -14,26 +14,24 @@
     "cucumber": "NODE_OPTIONS='--experimental-loader ts-node/esm' cucumber-js"
   },
   "dependencies": {
-    "@chakra-ui/react": "^1.8.8",
-    "@chakra-ui/system": "^2.1.1",
+    "@chakra-ui/react": ">=1.8.5 <=1.8.8",
+    "@chakra-ui/system": ">=1.11.0 <=1.12.1",
```

Comment:
why is there a max version for these?

Reply:
there were dependency issues that have since been resolved. Updated the package.json.